src/transaction/flashback_cl.c

```diff
@@ -164,21 +233,25 @@ flashback_unpack_and_print_summary (char **summary_buffer, FLASHBACK_SUMMARY_INF
 		}
 
 	      da_get (classname_list, idx, classname);
-	      printf ("%s ", classname);
+	      if (j == 0)
+		{
+		  printf ("%s\n", classname);
+		  line_cnt++;
+		}
+	      else
+		{
+		  printf ("%134s\n", classname);
```

Comment:
What's this format for?

Reply:
I'll modify the format so that the table names are left-aligned.